src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: `${name}`,
+    wheels: `${wheels}`,
+    version: `${version}`,
```

Comment:
```suggestion
    name,
    wheels,
    version,
```
Можна скористатись короткою формою запису

Reply:
Погоджуюся з Русланом